src/server/protocol.ts

```diff
@@ -598,6 +600,27 @@ export interface GetApplicableRefactorsResponse extends Response {
     body?: ApplicableRefactorInfo[];
 }
 
+/**
+ * Request refactorings at a given position or selection area to move to an existing file.
+ */
+export interface GetMoveToRefactoringFileSuggestionsRequest extends Request {
+    command: CommandTypes.GetMoveToRefactoringFileSuggestions;
+    arguments: GetMoveToRefactoringFileSuggestionsRequestArgs;
+}
+export type GetMoveToRefactoringFileSuggestionsRequestArgs = FileLocationOrRangeRequestArgs & {
+    kind?: string;
+};
+/**
+ * Response is a list of available files.
+ * Each refactoring exposes one or more "Actions"; a user selects one action to invoke a refactoring
+ */
+export interface GetMoveToRefactoringFileSuggestionsResponse extends Response {
+   body?: {
+       newFileName: string;
+      files: string[];
+   };

```

Comment:
Name this interface as `GetMoveToRefactoringFileSuggestions` or something like that.